mmseg/models/decode_heads/ic_head.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2021 Intel Corporation
```

Comment:
The original repo: https://github.com/hszhao/ICNet

Reply:
Ok